src/styles/index.css

```diff
@@ -12,7 +12,7 @@
 .grid-container { 
     display: grid;
     position: relative;
-    grid-template-columns: 250px 1fr;
+    grid-template-columns: 300px 1fr;
```

Comment:
please update the unit here as well

Reply:
The same reason as above - I used pixels here because I've used rem majorly for text. And since I have used rem for text, I've also decreased the font-size percentage of the root in smaller devices to make text look readable and user-friendly across all devices. 
Hence, this caused the width of the sections/ containers to decrease, which I did not want. Hence the reason to stick with pixels!